api/tests/lib/seed_e2e.py

```diff
@@ -0,0 +1,35 @@
+import os
+import tests.src.db.models.factories as factories
+import logging
+import uuid
+import src.adapters.db as db
+from src.auth.api_jwt_auth import create_jwt_for_user
+from src.auth.api_jwt_auth import initialize_jwt_auth
+from pathlib import Path
+from src.util.file_util import append_to_file
+
+logger = logging.getLogger(__name__)
+
+###############################
+# This script will
+# * seed the local database with a user that can be used for fake or potentially real logins from automated e2e tests
+# * create an authorization token entry tied to that user, seeding the database accordingly
+# * write the resulting token into the overrides.env file (where it can be retrieved by CI processes for frontend / e2e use)
+###############################
+
+
+def _append_token_to_override(token: str) -> None:
+    path_to_override = os.path.join(os.path.dirname(__file__), "..", "..", "override.env")
+    if Path(path_to_override).exists():
+        token_declaration = 'E2E_USER_AUTH_TOKEN="' + token + '"'
+        append_to_file(path_to_override, token_declaration)
+
+
+def _build_users_and_tokens(db_session: db.Session) -> None:
+    USER_E2E_BASE = factories.UserFactory.build(
+        user_id=uuid.UUID("7edb5704-9d3b-4099-9e10-fbb9f2729aff")
+    )
```

Comment:
Can you define this in the same pattern as the other user setup file? We'd additionally want to have at least a record in the link external user table so you could verify this works by logging into it.